Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -53,8 +53,11 @@ static void* s_stored_frame = nullptr;
 static u64 s_stored_frame_size = 0;
 static bool s_start_dumping = false;
 
-bool AVIDump::Start(HWND hWnd, int w, int h)
+bool AVIDump::Start(HWND hWnd, int w, int h, DumpFormat format)
 {
+	// Win32 only supports BGR format
```

Comment:
This comment is wrong, only this lines need to be touched: https://github.com/Sonicadvance1/dolphin/blob/ES_fix_framedump/Source/Core/VideoCommon/AVIDump.cpp#L289